### ISSUE TITLE
Peer warning/error after installing @nrwl/cli

### ISSUE THREAD

**Issue by ph55** [nrwl/nx#1680]
After installing `@nrwl/cli` globally, annoying warning shown:
```
 ✘ user@Users-MacBook-Pro > ~/Dev/corp/app > npm i -g @nrwl/cli
/usr/local/bin/nx -> /usr/local/lib/node_modules/@nrwl/cli/bin/nx.js
npm WARN @nrwl/tao@8.4.3 requires a peer of @nrwl/workspace@8.4.3 but none is installed. You must install peer dependencies yourself.

+ @nrwl/cli@8.4.3
updated 1 package in 8.531s
```

Also error shown after listing global packages:
```
 user@Users-MacBook-Pro > ~/Dev/corp/app > npm ls -g --depth=0
/usr/local/lib
├── @nativescript/schematics@0.6.0
├── @nrwl/cli@8.4.3
├── ios-deploy@1.9.2
├── nativescript@6.0.1
├── npm@6.10.2
├── npm-check-updates@3.1.1
├── pm2@3.4.0
├── tslint@5.12.0
└── typescript@3.2.2

npm ERR! peer dep missing: @nrwl/workspace@8.4.3, required by @nrwl/tao@8.4.3
```

**Comment by fernandosouza** [comment]
It also happened to me.

**Comment by vsavkin** [comment]
Thank you, folks!

Fixed it here: 166a21a27b91dfaf263ac843891ac22d1dcdcac8
